example/ck_tile/18_flatmm/run_flatmm_example.inc

```diff
@@ -18,7 +18,7 @@ constexpr const char* DataTypeToString()
     {
         return "bf8";
     }
-    else if  constexpr(std::is_same_v<T, ck_tile::bf16_t>)
+    else if constexpr(std::is_same_v<T, ck_tile::bf16_t>)
```

Comment:
Should we add `*.inc` to the clang-format CI script?
https://github.com/ROCm/composable_kernel/blob/c1badfd30c1679f4c8e176c8f0608db2c6ac6505/Jenkinsfile#L994-L1002

Reply:
maybe in a separate PR?